backend/fleet_management/models.py

```diff
@@ -125,3 +127,17 @@ def hash_form(initial_data: dict) -> int:
             "endMileage": initial_data["end_mileage"],
         }
         return hash_dict(required_fields)
+
+
+class Refuel(models.Model):
+    driver = models.ForeignKey(
+        User, on_delete=models.CASCADE, related_name="drives_refueled"
+    )
+    car = models.ForeignKey(Car, null=False, on_delete=models.CASCADE)
+    date = models.DateField(auto_now_add=True, blank=False)
+    current_mileage = models.IntegerField(null=False)
+    refueled_liters = models.IntegerField(null=False)
+    price_per_liter = models.IntegerField(null=False)
```

Comment:
```suggestion
@property
def price_per_liter(self):
    # TODO: add some checks which would wrap returned value to 2 decimal places
    return self.currency / self.refueled_liters
```
`price_per_liter` can be calculated from `refueled_liters` and `currency` fields.

I know that it was mentioned in ticket but I'm not sure if we really have to have separate column when we can derive it from other fields. That field would have to be filled by user which matters, when you have to fill lengthy form vs short 2-3 field form.